server/server.go

```diff
@@ -191,6 +191,29 @@ func New(cfg Config) (*Server, error) {
 	}, []string{"method", "route", "status_code", "ws"})
 	prometheus.MustRegister(requestDuration)
 
+	receivedMessageSize := prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: cfg.MetricsNamespace,
+		Name:      "request_message_bytes",
+		Help:      "Size (in bytes) of messages received in the request.",
+		Buckets:   middleware.BodySizeBuckets,
```

Comment:
Today I tried to use this metric, and found that all requests were in the smallest bucket of 1MB.
Was there some expectation that servers would receive massive requests?

Reply:
The idea of this work was to find out what the right limit for gRPC message sizes is. That's why we used relatively big buckets.